### DISCUSSION TITLE
Installing Rustdesk using Kaspersky Security Center with predefined settings

### DISCUSSION THREAD

**Discussion by elhananjair** [rustdesk/rustdesk#9241] [Q&A]
Greetings all
I have tried to install Rustdesk remotely using Kaspersky Security Center and have successfully done that. However since I am installing on standard users, I couldn't connect to the computer remotely through Rustdesk on LAN. To connect using LAN to the computer, the following parameters must be set on the remote Rustdest application; **Enable rdp session sharing** and **Enable direct IP access**. 
The user must be an administrator to enable those options, is there any parameters I can set on the installation package to allow those options before installation? just like other installation parameters, `/silent` ...

**Comment by rustdesk** [comment]
custom client  in`Pro` https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/

**Reply by elhananjair** [reply]
Do you mean, it's only available on the pro version? Therefore, is it not possible to set such a thing through the installation parameter?

**Reply by rustdesk** [reply]
> Therefore, is it not possible to set such a thing through the installation parameter?

Impossible, we won't add it for sure.
